# Hungarian translation of mintbackup.
# Copyright (C) 2013 Free Software Foundation, Inc.
# This file is distributed under the same license as the mintbackup package.
#
msgid ""
msgstr ""
"Project-Id-Version: mintbackup\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2010-04-20 10:28+0100\n"
"PO-Revision-Date: 2013-12-13 12:58+0000\n"
"Last-Translator: Frank Barnores <Unknown>\n"
"Language-Team: Hungarian\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2013-12-13 14:40+0000\n"
"X-Generator: Launchpad (build 16869)\n"
"Language: hu\n"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:33
#: usr/lib/linuxmint/mintBackup/mintBackup.py:98
#: usr/lib/linuxmint/mintBackup/mintBackup.py:238
#: usr/lib/linuxmint/mintBackup/mintBackup.py:398
#: usr/lib/linuxmint/mintBackup/mintBackup.py:468
#: usr/lib/linuxmint/mintBackup/mintBackup.py:484
#: usr/lib/linuxmint/mintBackup/mintBackup.py:541
#: usr/lib/linuxmint/mintBackup/mintBackup.py:593
#: usr/lib/linuxmint/mintBackup/mintBackup.py:623
#: usr/lib/linuxmint/mintBackup/mintBackup.py:637
#: usr/lib/linuxmint/mintBackup/mintBackup.py:644
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1464
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1554
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1562
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1602
msgid "Backup Tool"
msgstr "Mentéskezelő"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:34
msgid "Make a backup of your home directory"
msgstr "Mentés készítése a saját mappáról"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:154
msgid "Preserve structure"
msgstr "Struktúra megőrzése"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:156
msgid ".tar file"
msgstr ".tar fájl"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:157
msgid ".tar.bz2 file"
msgstr ".tar.bz2 fájl"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:158
msgid ".tar.gz file"
msgstr ".tar.gz fájl"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:164
msgid "Never"
msgstr "Soha"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:165
msgid "Size mismatch"
msgstr "Méreteltérés"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:166
msgid "Modification time mismatch"
msgstr "Módosítási idő eltérés"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:167
msgid "Checksum mismatch"
msgstr "Hibás ellenőrzőösszeg"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:168
msgid "Always"
msgstr "Mindig"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:190
msgid "Excluded paths"
msgstr "Kihagyott útvonalak"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:200
msgid "Type"
msgstr "Típus"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:204
msgid "Detail"
msgstr "Részletek"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:210
#: usr/lib/linuxmint/mintBackup/mintBackup.py:219
msgid "Path"
msgstr "Útvonal"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:213
#: usr/lib/linuxmint/mintBackup/mintBackup.py:222
msgid "Error"
msgstr "Hiba"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:258
msgid "Store?"
msgstr "Elmentsük?"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:261
#: usr/lib/linuxmint/mintBackup/mintBackup.py:274
msgid "Name"
msgstr "Név"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:271
msgid "Install"
msgstr "Telepítés"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:279
msgid "Backup or Restore"
msgstr "Biztonsági mentés vagy visszaállítás"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:280
msgid "Choose from the following options"
msgstr "Válasszon az alábbiak közül"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:282
#: usr/lib/linuxmint/mintBackup/mintBackup.py:298
#: usr/lib/linuxmint/mintBackup/mintBackup.py:312
#: usr/lib/linuxmint/mintBackup/mintBackup.py:319
#: usr/lib/linuxmint/mintBackup/mintBackup.py:323
#: usr/lib/linuxmint/mintBackup/mintBackup.py:328
msgid "Backup files"
msgstr "Fájlok biztonsági mentése"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:283
#: usr/lib/linuxmint/mintBackup/mintBackup.py:332
#: usr/lib/linuxmint/mintBackup/mintBackup.py:342
#: usr/lib/linuxmint/mintBackup/mintBackup.py:348
#: usr/lib/linuxmint/mintBackup/mintBackup.py:353
msgid "Restore files"
msgstr "Fájlok visszaállítása"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:284
#: usr/lib/linuxmint/mintBackup/mintBackup.py:357
#: usr/lib/linuxmint/mintBackup/mintBackup.py:362
#: usr/lib/linuxmint/mintBackup/mintBackup.py:368
#: usr/lib/linuxmint/mintBackup/mintBackup.py:373
msgid "Backup software selection"
msgstr "Szoftverválaszték mentése"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:285
#: usr/lib/linuxmint/mintBackup/mintBackup.py:377
#: usr/lib/linuxmint/mintBackup/mintBackup.py:382
#: usr/lib/linuxmint/mintBackup/mintBackup.py:389
msgid "Restore software selection"
msgstr "Szoftverválaszték visszaállítása"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:287
msgid "Make a backup of your files"
msgstr "Saját fájlok biztonsági mentése"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:288
msgid "Save the list of installed applications"
msgstr "Telepített alkalmazások listájának mentése"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:289
msgid "Restore a previous backup"
msgstr "Korábbi mentés visszaállítása"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:290
msgid "Restore previously installed applications"
msgstr "Korábban telepített alkalmazások visszaállítása"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:299
msgid "Please select a source and a destination for your backup"
msgstr "Válassza ki a biztonsági mentés forrását és célját"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:300
#: usr/lib/linuxmint/mintBackup/mintBackup.py:336
#: usr/lib/linuxmint/mintBackup/mintBackup.py:344
msgid "Source:"
msgstr "Forrás:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:301
#: usr/lib/linuxmint/mintBackup/mintBackup.py:337
msgid "Destination:"
msgstr "Cél:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:302
#: usr/lib/linuxmint/mintBackup/mintBackup.py:338
msgid "Advanced options"
msgstr "Haladó beállítások"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:303
#: usr/lib/linuxmint/mintBackup/mintBackup.py:345
msgid "Description:"
msgstr "Leírás:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:304
msgid "Output:"
msgstr "Kimenet:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:305
#: usr/lib/linuxmint/mintBackup/mintBackup.py:339
msgid "Overwrite:"
msgstr "Felülírás:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:306
msgid "Confirm integrity"
msgstr "Integritás megőrzése"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:307
msgid "Follow symlinks"
msgstr "Szimbolikus linkek követése"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:308
msgid "Preserve permissions"
msgstr "Jogosultságok megőrzése"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:309
msgid "Preserve timestamps"
msgstr "Időbélyegek megőrzése"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:313
msgid "Please select any files or directories you want to exclude"
msgstr "Válassza ki a kihagyni kívánt fájlokat vagy könyvtárakat"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:314
msgid "Exclude files"
msgstr "Kihagyandó fájlok"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:315
msgid "Exclude directories"
msgstr "Kihagyandó mappák"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:316
msgid "Remove"
msgstr "Eltávolítás"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:320
msgid "Please review the information below before starting the backup"
msgstr "Nézze át a beállításokat a mentés megkezdése előtt"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:324
msgid "Please wait while your files are being backed up"
msgstr "Kérem várjon, amíg a mentés befejeződik"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:325
#: usr/lib/linuxmint/mintBackup/mintBackup.py:370
msgid "Backing up:"
msgstr "Biztonsági mentés:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:329
#: usr/lib/linuxmint/mintBackup/mintBackup.py:374
msgid "The backup is now finished"
msgstr "A biztonsági mentés befejeződött"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:333
msgid ""
"Please choose the type of backup to restore, its location and a destination"
msgstr "Válassza ki a visszaállítandó mentés típusát, helyét és célját"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:334
msgid "Archive"
msgstr "Archívum"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:335
msgid "Directory"
msgstr "Könyvtár"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:343
msgid "Please review the information below"
msgstr "Nézze át az alábbiakat"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:349
msgid "Please wait while your files are being restored"
msgstr "Kérem várjon, amíg a visszaállítás befejeződik"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:350
msgid "Restoring:"
msgstr "Visszaállítás:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:354
msgid "The restoration of the files is now finished"
msgstr "A fájlok visszaállítása befejeződött"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:358
msgid "Please choose a destination"
msgstr "Válassza ki a célhelyet"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:359
#: usr/lib/linuxmint/mintBackup/mintBackup.py:557
msgid "Destination"
msgstr "Cél"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:363
msgid "The list below shows the packages you added to Linux Mint"
msgstr ""
"Az alábbi listán azokat a csomagokat találja, amelyeket Ön adott a Linux "
"Mint-hez"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:364
#: usr/lib/linuxmint/mintBackup/mintBackup.py:384
msgid "Select all"
msgstr "Összes kijelölése"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:365
#: usr/lib/linuxmint/mintBackup/mintBackup.py:385
msgid "Deselect all"
msgstr "Kijelölés megszüntetése"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:369
msgid "Please wait while your software selection is being backed up"
msgstr "Kérem várjon, amíg a szoftverkészlet mentésre kerül"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:378
msgid "Please select a saved software selection"
msgstr "Válasszon egy elmentett szoftverkészletet"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:379
msgid "Software selection:"
msgstr "Szoftverkészlet:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:383
msgid "Select the packages you want to install"
msgstr "Válassza ki a telepíteni kívánt csomagokat"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:386
msgid "Refresh"
msgstr "Frissítés"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:390
msgid "The restoration is now finished"
msgstr "A visszaállítás befejeződött"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:391
msgid "Your package selection was restored succesfully"
msgstr "A csomagkészlet sikeresen visszaállításra került"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:396
msgid "About"
msgstr "Névjegy"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:477
#: usr/lib/linuxmint/mintBackup/mintBackup.py:493
msgid "Invalid path"
msgstr "Érvénytelen útvonal"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:477
#: usr/lib/linuxmint/mintBackup/mintBackup.py:493
#, python-format
msgid "%s is not located within your source directory. Not added."
msgstr "%s nem található a forráskönyvtárban. Nem lett hozzáadva."

#: usr/lib/linuxmint/mintBackup/mintBackup.py:541
msgid ""
"Please choose different directories for the source and the destination"
msgstr "Válasszon különböző könyvtárat a forrásnak és a célnak"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:556
msgid "Source"
msgstr "Forrás"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:559
msgid "Description"
msgstr "Leírás"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:563
msgid "Compression"
msgstr "Tömörítés"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:567
msgid "Overwrite destination files"
msgstr "Létező fájlok felülírása"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:570
msgid "Exclude"
msgstr "Kihagyás"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:593
msgid "Please choose a file to restore from"
msgstr "Válassza ki a fájlt, amelyből vissza szeretne állítani"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:623
msgid "Please choose a destination directory"
msgstr "Válassza ki a célkönyvtárat"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:637
msgid "No packages need to be installed at this time"
msgstr "Nem került telepítése csomag"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:644
msgid "Please select one or more packages to install"
msgstr "Válassza ki a telepítendő csomagokat"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:703
#: usr/lib/linuxmint/mintBackup/mintBackup.py:704
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1180
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1182
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1477
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1478
msgid "Calculating..."
msgstr "Számítás…"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:753
#: usr/lib/linuxmint/mintBackup/mintBackup.py:793
msgid "Skipping broken link"
msgstr "Törött linkek kihagyása"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:754
msgid "Broken link"
msgstr "Törött link"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:757
#: usr/lib/linuxmint/mintBackup/mintBackup.py:797
msgid "Skipping link"
msgstr "Link kihagyása"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:821
#: usr/lib/linuxmint/mintBackup/mintBackup.py:830
#: usr/lib/linuxmint/mintBackup/mintBackup.py:839
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1242
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1253
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1266
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1331
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1340
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1350
msgid "Skipping identical file"
msgstr "Azonos fájl kihagyása"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:861
#, python-format
msgid ""
"Warning: Some files were not saved, copied: %(current_file)d files out of "
"%(total)d total"
msgstr ""
"Figyelmeztetés: Néhány fájl nem lett mentve, %(total)d fájlból "
"%(current_file)d fájl lett átmásolva"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:865
msgid "An error occured during the backup"
msgstr "A mentés közben hiba lépett fel"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:875
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1520
msgid "The backup was aborted"
msgstr "Mentés megszakítva"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:883
msgid "The backup completed successfully"
msgstr "A mentés sikeresen befejeződött"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:968
msgid "Checksum Mismatch:"
msgstr "Ellenőrzőösszeg hiba:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:969
msgid "Checksum Mismatch"
msgstr "Ellenőrzőösszeg hiba"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1019
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1021
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1048
msgid "Calculating checksum"
msgstr "Ellenőrzőösszeg számítása"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1090
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1114
msgid "(Not created with the Backup Tool)"
msgstr "(nem a Mentéskezelővel készült)"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1374
#, python-format
msgid ""
"Warning: Some filed were not restored, copied: %(current_file)d files out of "
"%(total)d total"
msgstr ""
"Figyelmeztetés: Néhány fájl nem lett visszaállítva, %(total)d fájlból "
"%(current_file)d fájl lett átmásolva"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1377
msgid "An error occured during the restoration"
msgstr "A visszaállítás közben hiba lépett fel"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1388
msgid "The restoration was aborted"
msgstr "Visszaállítás megszakítva"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1396
msgid "The restoration completed successfully"
msgstr "A visszaállítás sikeresen befejeződött"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1511
msgid "An error occured during the backup:"
msgstr "A mentés közben hiba lépett fel:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1528
msgid "Your software selection was backed up succesfully"
msgstr "A szoftverkészlet mentése sikerült"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1554
msgid "The specified file is not a valid software selection"
msgstr "A megadott fájl nem érvényes szoftverkészlet"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1562
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1602
msgid "An error occurred while accessing the file"
msgstr "Hiba történt a fájl elérése közben"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1595
msgid "Could not locate the package"
msgstr "A csomag nem található"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1626
msgid "Please wait, this can take some time"
msgstr "Kis türelmet, ez eltarthat egy ideig"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1628
msgid "The installation is complete"
msgstr "A telepítés kész"

#~ msgid "Archiving your home directory..."
#~ msgstr "A Saját mappa archiválása..."

#~ msgid "Backup successful"
#~ msgstr "A mentés sikerült"

#~ msgid "An error occurred during the backup:"
#~ msgstr "A következő hiba lépett fel a mentés során:"

#~ msgid "Backup failed"
#~ msgstr "A mentés nem sikerült"

#~ msgid "Extracting the content of the backup into your home directory..."
#~ msgstr "A mentés tartalmának kicsomagolása a Saját mappába..."

#~ msgid "Copying the backup file into your home directory..."
#~ msgstr "A mentésfájl másolása a Saját mappába..."

#~ msgid "Restoration successful"
#~ msgstr "A visszaállítás sikerült"

#~ msgid "Cleaning up..."
#~ msgstr "Takarítás..."

#~ msgid "Your backup was successfully restored"
#~ msgstr "A mentés sikeresen visszaállítva"

#~ msgid "Your home directory was successfully backed-up into"
#~ msgstr "A Saját mappa sikeresen mentve a következő fájlba:"

#~ msgid "Backup"
#~ msgstr "Mentés"

#~ msgid "Restoration failed"
#~ msgstr "A visszaállítás nem sikerült"

#~ msgid "An error occurred while restoring the backup archive:"
#~ msgstr "A következő hiba lépett fel a visszaállítás során:"

#~ msgid "Read error"
#~ msgstr "Olvasási hiba"

#~ msgid "Restore your personal data from this backup"
#~ msgstr "Személyes adatainak visszaállítása ebből a mentésből"

#~ msgid "<big><b>Load data into your home directory</b></big>"
#~ msgstr "<big><b>Adatok visszatöltése a Saját mappába</b></big>"

#~ msgid "is not located within your home directory. Not added."
#~ msgstr "nem a Saját mappában található. Nem adható meg."

#~ msgid "Included hidden directories"
#~ msgstr "Mentendő rejtett fájlok és mappák"

#~ msgid "Included"
#~ msgstr "Mentés"

#~ msgid "Restore"
#~ msgstr "Visszaállítás"

#~ msgid "Overwrite existing files"
#~ msgstr "Létező fájlok felülírása"

#~ msgid "View content"
#~ msgstr "Tartalom megtekintése"

#~ msgid "Exclude folders"
#~ msgstr "Mappák kihagyása"

#~ msgid "Excluded Files and Directories"
#~ msgstr "A mentésből kihagyott fájlok és mappák"

#~ msgid "Hidden paths"
#~ msgstr "Rejtett elemek"

#~ msgid "Opening the backup archive..."
#~ msgstr "A mentési archívum megnyitása..."

#~ msgid "An error occurred while opening the backup:"
#~ msgstr "A következő hiba lépett fel a mentés megnyitása során:"
